arithmetic_analysis/newton_backward_interpolation.py

```diff
@@ -0,0 +1,55 @@
+# https://www.geeksforgeeks.org/newton-forward-backward-interpolation/
+import math
+
+
+# for calculating u value
+def ucal(value_of_u: float, value_of_n: int) -> float:
+    """
+    >>> ucal(1, 2)
+    2
+    >>> ucal(1.1, 2)
+    2.5410000000000004
+    >>> ucal(1.2, 2)
```

Comment:
If you add the range and type check, you can add som doctests that error with "incorrect type", etc